frontend/src/components/Comments/CommentItem.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { SubcommentItem } from './SubcommentItem.js';
+import { NewComment } from './NewComment.js';
+
+import { formatDate } from '../../wrappers/formatDate.js';
+
+export const CommentItem = ({ comment, article, setArticle }) => {
+  const [visible, setShowResults] = useState(false);
+  const showNewComment = () => {
+    visible ? setShowResults(false) : setShowResults(true);
```

Comment:
jeęsli visible jest true, to ustawiasz showResults na false?

Reply:
Bo to jest toggle. Po wywołaniu showNewComment odwraca true/false.